doc/DEPENDENCIES.md

```diff
@@ -163,4 +165,20 @@ If the above `update` command does not check out the expected version, this can
 
 The version used is automatically handled by submodule config.  This can be synced by re-running the initialization command above.
 
-As of project version `0.1.0`, the required version is tag `v2.6.0`.
\ No newline at end of file
+As of project version `0.1.0`, the required version is tag `v2.6.0`.
+
+## The `dmod.subsetservice` Package
+
+This is an optional package available via the [OWP DMOD tool](https://github.com/NOAA-OWP/DMOD).  It only is required to enable the driver to generate partition-specific subdivided hydrofabric files from the complete hydrofabric file, as is conditionally required for [distributed processing](DISTRIBUTED_PROCESSING.md).  Further, its use is only attempted when such files are needed for distributed processing _and_ are not already created.
+
+As such, this dependency is ignored unless both Python support and MPI support are enabled and built into the _ngen_ driver.
+
+### Setup
+
+This must be installed in the active Python environment at the time the `ngen` executable is called.  This is usually best done via a virtual environment, especially if one is already set up as described for the [Numpy](#numpy) dependency.  However, all that matters is that the Python environment available to the ngen driver has this package installed.
+
+If installing to a virtual env as suggested above, the installation steps are similar to what is described in the [DMOD project document for the subsetting CLI tool]( https://github.com/NOAA-OWP/DMOD/tree/master/doc/SUBSETTING_CLI_TOOL.md).  The difference is just to create and/or source a virtual environment locate here first, and then change to the DMOD project directory.  All the example commands after the step of sourcing the virtual environment can then be followed as described on that page.
+
+### Version Requirements
+
+Version `0.3.0` or greater must be installed, which is available after approval of DMOD PR #92.
```

Comment:
This PR has been merged, I would remove the second part of this sentence alltogether.